tests/test_project/tests/test_password_history.py

```diff
@@ -100,3 +100,179 @@ def test_change_number_hasher_iterations(self):
                     user_config__iterations=HistoryVeryStrongHasher.iterations).count(),
                 1,
             )
+
+    @override_settings(AUTH_PASSWORD_VALIDATORS=[{
+        'NAME': 'django_password_validators.password_history.password_validation.UniquePasswordsValidator',
+        'OPTIONS': {
+            'lookup_range': 5
+        }
+    }])
+    def test_lookup_range_true_positive(self):
```

Comment:
@spokerman12, same as [this comment](https://github.com/fizista/django-password-validators/pull/18/files#r566358291).

Also can you add a docstring for the test functions that you have added?

Reply:
Sure thing.